### ISSUE TITLE
Read CSV files in radiative heat transfer applications (weather information)

### ISSUE THREAD

**Issue by lberti** [feelpp/feelpp#2043]
It should be possible to specify CSV files for 
- black body temperature (sky temperature, if the Sun is the black body)
- incident radiation (incident solar radiation, if the Sun is the radiative source)
in radiative heat transfer applications.

**Comment by prudhomm** [comment]
@lberti @Zohra-D I imagine that this type of information can come from

 - CSV file as you said
 - online databases 

would it make sense to also consider the latter case ?

**Comment by lberti** [comment]
If it came from online databases, would it need to be preprocessed/downloaded before?
In this issue, I was thinking to address the addition of Json blocks to account for time-varying quantities. The preprocessing part (i.e. computing these quantities from the available data) should be done separately, using tools developed for 
- #1906
-  #1907
Or am I misunderstanding the comment?

**Comment by Zohra-D** [comment]
> The preprocessing part (i.e. computing these quantities from the available data) should be done separately, using tools developed for
> 
> * [add sky temperature models #1906](https://github.com/feelpp/feelpp/issues/1906)
> * [compute solar flux on surfaces #1907](https://github.com/feelpp/feelpp/issues/1907)
>   Or am I misunderstanding the comment?

Yes, this can be done using the ktirio tool [weather](https://github.com/feelpp/ktirio/tree/develop/ibat/tools/weather)

**Comment by Zohra-D** [comment]
> If it came from online databases, would it need to be preprocessed/downloaded before?

The sky temperature may not be directly available, so we need to calculate it. In addition, we need to calculate the incident solar flux on our building surfaces.

**Comment by lberti** [comment]
@prudhomm I think adding these to the Json structure created for radiative heat transfer, which would mean adding further commits to PR #2039. I propose to merge the PR before I start working on this.

**Comment by prudhomm** [comment]
@lberti where do we stand on this one ?
